### ISSUE TITLE
Exception in server tick loop - Game freezes randomly and does not come back 

### ISSUE THREAD

**Issue by AndreCelso** [sddsd2332/Mekanism-CE-Unofficial-1.12.2#74]
Crash log: 
[crash-2024-09-17_14.49.24-server.txt](https://github.com/user-attachments/files/17032953/crash-2024-09-17_14.49.24-server.txt)

**Comment by sddsd2332** [comment]
emm
I don't see what's going wrong
Are you switching directly from the original mek to mekceu? 
Also, the mekceu does not support AEAdditions, switch to the secondary MekanismEnergistics of AE2UEL
[If you need AE components above 64K, you can try NAE2]

**Comment by AndreCelso** [comment]
no, i use mekceu for a long time, i guess since it was on beta, i do not remember if i submitted an issue here for the auto-sorting suddenly stop working back then, but i told Arain about it. About AEAdditions, i already do use Mekanism Energistics for the Gas Storage, as i use AE2 Fluid Crafting (another mod from GlodBlock as well) to work with Gas Crafting via AE2. i just did not get rid of it because of other features i judged i needed them, i don't remember now if it was its high tier Crafting Storages Units (CPUs). If you're ARain, you probably will remember of me from AstroTibs Discord Server, the server for the Modernity Project, and i tried to find a solution for the auto-sorting not working with you back then, even sent you my logs

**Comment by AndreCelso** [comment]
i'm really desperate to find a solution to this crash, as i do not want to lose my modpack/world i spent so many hours fostering. You're one of the few people who can help me, as i do not have the slightest clue on why is this happening, what's causing it. I even tried to ask for help on mekanism discord server, but it didn't solve it

**Comment by AndreCelso** [comment]
i also use Neeve's AE2

**Comment by sddsd2332** [comment]
Did you do anything with the conduit before the server crashed?

**Comment by AndreCelso** [comment]
conduit? what do you mean? you mean the EnderIO Conduit?

**Comment by sddsd2332** [comment]
It could be MEK's pipeline, or it's possible that there was a problem with EIO's gasconduits

**Comment by AndreCelso** [comment]
ah i do have GasConduits on the modpack, tho i did not use it on the world that's crashing yet. i swear, i did not even touch the gas conduits yet, as i storage and deal with gases mostly via AE2 with Mekeng and AE2 Fluid Crafting Rework

**Comment by sddsd2332** [comment]
I don't know
If it really doesn't work, use nbtexplorer to remove all mek pipes from the map to try to fix it

**Comment by AndreCelso** [comment]
i tried to break every mek pipe i could think of i placed in the world, as soon as i broke the last of them (i guess) some weird toast started popping in, like the "find a tree and punch it" toast and "craft planks" toast. Is that somehow related? is that a symptom of things being fixed?

**Comment by AndreCelso** [comment]
tested the world for about 30 minutes after i broke all the mekanism pipes i had placed on my world, and it did not crash. It was crashing a lot easier and faster than that before. Not sure if it was mekanism faulty, nor if the issue was truly fixed, but i leave my feedback here, in case it's useful to you somehow

**Comment by sddsd2332** [comment]
Shut down
If there are still issues with the pipeline, reopen a new issue
